src/commands/createAccount.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import cryptoModule from '../utils/cryptoModule';
+import config from '../utils/env';
+import { createMnemonicPassphrase } from '../utils/mnemonic';
+import { printResult } from '../utils/print';
+import commonOptions from '../utils/options';
+import {
+	shouldUseJsonOutput,
+} from '../utils/helpers';
+
+const description = `Create account returns a new created passphrase with associate publicKey and accountId.
```

Comment:
`randomly-generated mnemonic passphrase with its corresponding public key and account ID`